planner/core/common_plans.go

```diff
@@ -133,6 +133,13 @@ type CancelDDLJobs struct {
 	JobIDs []int64
 }
 
+// Change represents a change plan.
+type Change struct {
+	baseSchemaProducer
+
+	Statement *ast.ChangeStmt
```

Comment:
Ditto

Reply:
ChangeStmt is used to update the statement metrics counter.

Reply:
This will make the code more concise

Reply:
I think we can define it as :
type Change struct {
    baseSchemaProducer
    *ast.ChangeStmt
}

Thus we can use e.xxx to without `stmt := e.Statement`